chromium/browser_type.go

```diff
@@ -383,19 +383,19 @@ func execute(
 		// TODO: How to handle these errors?
 		defer func() {
 			if err := dataDir.Cleanup(); err != nil {
-				logger.Errorf("BrowserType:execute", "%v", err)
+				logger.Errorf("closing the browser", "%v", err)
 			}
 		}()
 
 		if err := cmd.Wait(); err != nil {
-			log := logger.Errorf
 			if err.Error() == "signal: killed" || err.Error() == "exit status 1" {
 				// The browser process is killed when the context is cancelled
 				// after a k6 iteration ends, so silence the log message until
 				// we can stop it gracefully. See #https://github.com/grafana/xk6-browser/issues/423
-				log = logger.Debugf
+				logger.Errorf("closing the browser", "process with PID %d unexpectedly ended: %v", cmd.Process.Pid, err)
+			} else {
+				logger.Debugf("BrowserType:Execute", "process with PID %d unexpectedly ended: %v", cmd.Process.Pid, err)
```

Comment:
Indecisive about this done. I thought `BrowserType:Execute` would be meaningless to users.

Reply:
> in which case BrowserType:Execute is fine

I was talking about the category of `Errorf`.

> "browser" makes sense to me.

OK, that makes sense to me too.

> Note that we are potentially losing the method that logged the message, but hopefully that should be clear from the message itself.

Most are there. We can search and find them on an error.